### PR TITLE
gxmessage: init at 3.4.3

### PR DESCRIPTION
**Pull request by tftio**
A dropin replacement for xmessage, built against GTK+3.

### CODE REVIEW

**Comment by jfb** [comment]
Is there anything I need to do here? The package builds and installs successfully locally, for what it's worth.

File: pkgs/applications/misc/gxmessage/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{stdenv, fetchurl, gnome3, intltool, pkgconfig, texinfo}:
+
+stdenv.mkDerivation rec {
+  name = "gxmessage";
+  version = "3.4.3";
+
```
Comment (joachifm) [inline pkgs/applications/misc/gxmessage/default.nix]: Usually, the name contains the version, as in "foo-${version}" or inline.

Comment (jfb) [inline pkgs/applications/misc/gxmessage/default.nix]: I'll modify it, but is there any reason to push until the travis-ci problem is addressed?

Comment (joachifm) [inline pkgs/applications/misc/gxmessage/default.nix]: I fail to see how the travis build failure could be related.

Comment (jfb) [inline pkgs/applications/misc/gxmessage/default.nix]: Well, sure. But won't pushing a new commit trigger a travis job? That failure in travis looks unrelated to this PR, so I guess there's no reason not to push.

Comment (joachifm) [inline pkgs/applications/misc/gxmessage/default.nix]: Don't worry too much about travis, the build is quite unreliable. You'll probably be asked to squash your commits, but other than that it looks good to merge to me.

**Comment by jfb** [comment]
It's ready to go, thanks @joachifm

**Comment by lethalman** [comment]
Built locally thanks.

**Pull request merged**